webapp/portlet/src/main/webapp/space-members/components/SpaceInvitationDrawer.vue

```diff
@@ -161,6 +161,9 @@ export default {
     },
     defaultAvatar() {
       return `${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/users/default-image/avatar`;
+    },
+    invitedUser() {
+      return this.invitedMembers.length === 1 ? this.invitedMembers[0].profile.fullName : 'users';
     }
```

Comment:
You can add two computed variables one for the `invitedUser` object and the other for the `fullName` .